pages/common/keepassxc-cli.md

```diff
@@ -0,0 +1,36 @@
+# keepassxc-cli
+
+> Command-line interface for KeepassXC.
+> More information: <https://keepassxc.org/>.
+
+- Search entries:
+
+`keepassxc-cli lookup {{/path/to/database_file}} {{name}}`
```

Comment:
```suggestion
`keepassxc-cli lookup {{path/to/database_file}} {{name}}`
```
Please remove the leading slash if relative paths are also allowed.

Reply:
Will do